nilearn/_utils/niimg_conversions.py

```diff
@@ -138,7 +223,8 @@ def concat_niimgs(niimgs, dtype=np.float32, accept_4d=False,
         the dtype of the returned image
 
     accept_4d: boolean, optional
-        Accept 4D images
+        Also parse niimg-like objects in niimgs when one of them reflects
+        more than one image (i.e., has a time dimension)
```

Comment:
accept_4d is not used in this function !

Not sure what you mean with this docstring. Isn't it just that you can concatenate 4D niimg-like images together along the 4th dimension.